### ISSUE TITLE
Existing raindrops do not import on single-page mode

### ISSUE THREAD

**Issue by cjfagerstrom** [phildenhoff/logseq-raindrop#83]
I have a number of raindrops and can add them manually in multi-page mode, but in single-page mode it says sync complete without importing any existing raindrops. If I create a new raindrop and hit sync it will import my new raindrop.

**Comment by phildenhoff** [comment]
Have you tried setting the sync date to a date a few years ago (before you used Raindrop)? That should ensure everything is synced from your account.

----

Possible bug: maybe sync date is set in multi-page mode, without syncing anything to Logseq? Need to double-check transition from multi-page to single-page mode.

**Comment by cjfagerstrom** [comment]
Ah - missed where it says I can clear the timestamp to reimport everything. I'll test this.

**Comment by cjfagerstrom** [comment]
Adjusting the timestamp does work - I just missed the note on it and was expecting it to import all by default on the first import. Ideally, it would leave the field blank or have a toggle for import all vs after this timestamp date.

**Comment by phildenhoff** [comment]
Yes that makes perfect sense, so I'm going to reopen & update your issue to describe that expected behaviour. You're 100% right :)
